tests/console/journalctl.pm

```diff
@@ -136,6 +135,7 @@ sub run {
 
     # Write first series of messages with different log priority
     write_test_log_entries \%log_entries;
+    sleep 5;
```

Comment:
This is good for now, but I would rather aim at doing a `script_retry` grepping for the last entry a couple of times. What do you think?

Reply:
yup, forgot about it. I will add it

Reply:
It's there. Thanks!